invenio_app_ils/documents/jsonresolvers/document_eitem.py

```diff
@@ -27,18 +28,30 @@ def eitems_resolver(document_pid):
         eitem_search = current_app_ils.eitem_search_cls()
         for hit in eitem_search.search_by_document_pid(document_pid).scan():
             eitem = hit.to_dict()
-            eitems.append({
-                "pid": eitem.get("pid"),
-                "description": eitem.get("description"),
-                "internal_notes": eitem.get("internal_notes"),
-                "open_access": eitem.get("open_access"),
-                "bucket_id": eitem.get("bucket_id", None),
-                "files": eitem.get("files", []),
-            })
-        return {
-            "total": len(eitems),
-            "hits": eitems
-        }
+            files = []
+            for obj in EItem.get_record_by_pid(eitem.get("pid")).files:
```

Comment:
I can't think of something else because the files are empty in the eitem from the `eitem_search` but they are resolved in the `EItem.get_record_by_pid`.

Reply:
it means most likely that the files are not indexed for some reason